docs/user_guide/federated_xgboost/secure_xgboost_user_guide.rst

```diff
@@ -78,6 +78,8 @@ The following docker image is recommended for GPU training:
 
 Most Linux distributions are supported, as long as they have a recent glibc. The oldest glibc version tested is 2.35. Systems with older glibc may run into issues.
 
+.. _provisioning:
+
 NVFlare Provisioning
```

Comment:
we already have this defined in `./docs/programming_guide/provisioning_system.rst:.. _provisioning:`

would this work?

Reply:
The local reference would be used before the provisioning_system.rst one

Reply:
thanks, good to learn that.

Reply:
Actually I just tested myself to confirm, the local link does work as intended, however now other :ref:`provisioning` in other pages link to this xgboost section. We need rename this tag.

Reply:
@nvidianz

Reply:
Renamed to xgb_provisioning.